### ISSUE TITLE
Option to Disable image repo download

### ISSUE THREAD

**Issue by NS57** [SkyblockerMod/Skyblocker#679]
### Describe the new feature.

Option to opt out from downloading the image-repo folder into the config folder, as it adds a large amount of 
unnecessary bloat and won't be used by some users

**Comment by AzureAaron** [comment]
> ### Describe the new feature.
> Option to opt out from downloading the image-repo folder into the config folder, as it adds a large amount of unnecessary bloat and won't be used by some users
> 
> ### Images
> _No response_

The image repo right now is unused but in the future we will use it to add images to the mod's config.

**Comment by NS57** [comment]
Makes sense, Right now i assumed it was a really hidden way to show the mod's features
